DailySelfie/app/src/main/java/com/teamagam/dailyselfie/LoadPictureTask.java

```diff
@@ -0,0 +1,75 @@
+package com.teamagam.dailyselfie;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.BitmapFactory.Options;
+import android.os.AsyncTask;
+import android.widget.ImageView;
+
+import java.lang.ref.WeakReference;
+
+class LoadPictureTask extends AsyncTask<String, Void, Bitmap> {
+    private final Context mContext;
+    private final WeakReference<ImageView> mImageViewReference;
+
+    LoadPictureTask(Context context, ImageView imageView) {
+        // Use a WeakReference to ensure the ImageView can be garbage collected
+        mContext = context;
+        mImageViewReference = new WeakReference<>(imageView);
```

Comment:
why do use weakreference for the imageview.
but not for the context.
also you can get the context from the view